INTEGRATION.md

```diff
@@ -0,0 +1,55 @@
+# How to Integrate Grill.chat to your app 🤝
+
+This guide will help you to integrate Grill.chat to your app, with all the options available.
+
+## Iframe Integration
+
+The easiest way to integrate Grill.chat to your app is to use the iframe integration. It's a simple HTML tag that you can add to your app, and it will render the chat in an iframe.
+
+### 1. Add Iframe tag to your app
+
+Add the following HTML tag to your app, where you want to render the chat and style it however you want.
+
+```html
+<iframe allow="clipboard-write" src="https://x.grill.chat"></iframe>
+```
+
+### 2. Customize the `src` link
+
+The `src` link can be customized to your needs. Below includes the list of options that you can use to customize the chat.
+
+#### 2.1. Space Options
+
+Grill.chat home page contains list of topics for user to choose from. This topics are listed from all the posts in a subsocial space.
+
+For example, if you create a space with ID `1002`, then you can set the iframe `src` to `https://grill.chat/1002` to only show topics from that space.
```

Comment:
```suggestion
For example, if you create a space with the ID `1002`, you can set the iframe `src` to `https://grill.chat/1002` and it will only show the topics from that space.
```